### PR TITLE
Gaj/feature/static login

### LINKED ISSUE #18: Static Login Page Component
As a user, I should be able to see a page that has a space for login. 

It is not necessary for this page to be functional for MVP

### PR DESCRIPTION
**Pull request by gaj23**
#### Purpose:
 > Check all that apply.
- [x] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [ ] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 

This adds a static login page with basic logic for checking user login.

Issues closed #18 

NA

> Describe your code changes in detail for reviewers.

Login component was included to prepare for application of authentication. For now, is basic and will allow a user to login with any combo of username or password as long as both are more than 6 characters long.

See `components/Login` for styling and logic. 

Please run `npm start` on this branch and check that in `App` that Login is imported and is being called. 

Apply Route to Login.

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  //default is true! So no error message appears
+  //if value is blank, then setValidLogin(false)
+  //display error message
+  //include minvalue for security
+
+  const checkLogin = (event) => {
+    if ( username.length > 6 && password.length > 6 ) {
+      event.preventDefault();
+      setValidLogin(false);
```
Comment (kelsiebesingeryeh) [inline src/components/Login/Login.js:20]: why would the setValidLogin change to false here?

File: src/components/Login/Login.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+@import '../../_variables.scss';
```
Comment (JeffKersting) [inline src/components/Login/Login.scss:1]: Really like the styling on the form.

**Review by kelsiebesingeryeh — APPROVED** [review]

File: src/components/Login/Login.js (hunk 1 above)
Comment (gaj23) [reply src/components/Login/Login.js:20]: This was edited to demonstrate the UI for if a user entered in the incorrect info. I'll need to change it back. Great catch, thank you.

File: src/components/Login/Login.scss (hunk 2 above)
Comment (gaj23) [reply src/components/Login/Login.scss:1]: Sweet, maybe it's something we can continue in the rest of the forms, then?

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      console.log(validLogin)
+
```
Comment (JeffKersting) [inline src/components/Login/Login.js:22]: For MVP, will we be sending a request to the BE on login check? What will the query look like (send user name and get user data as response?), or verify username existence and query user data on user landing page load?

**Review by JeffKersting — APPROVED** [review]

**Pull request merged**